packages/trimerge-sync/src/TrimergeClient.ts

```diff
@@ -24,6 +22,14 @@ type AddCommitType =
   // Added from this client, but don't sync to store
   | 'temp';
 
+export type SubscribeEvent = {
+  origin:
+    | 'subscribe' // We send an initial event when you first subscribe
```

Comment:
trimerge-sync?

Reply:
store?

Reply:
As in when you call `client.subscribeDoc(callback)` it immediately calls `callback(currentDoc)` — though while working on this PR I questioned whether that's the right behavior (but didn't want to change in this PR)

Reply:
Oh, yeah I mean, I guess my main thing is that subscribe doesn't feel like an origin, I felt like maybe we `store` would better represent what the origin of this doc is

Reply:
that's not strictly right either. The source of that value could be self, local, or remote. Maybe another way to think of the origin (ie who triggered the callback) is "initial value" or init/initialize

Reply:
gonna merge for now, let's revisit whether this initial event should even be triggered at all